packages/api/shared/mailer.ts

```diff
@@ -1,16 +1,78 @@
 import { createTransport, SendMailOptions } from 'nodemailer';
+import { EmailStatus } from '../prisma/client';
+import { auth } from './auth';
 
-const transporter = createTransport({
-  url: process.env['EMAIL_SERVER'],
-});
+if (!process.env['EMAIL_SERVER'] && process.env.NODE_ENV === 'production') {
+  throw new Error('missing EMAIL_SERVER environment variable');
+}
+
+const transporter = process.env['EMAIL_SERVER']
+  ? createTransport({
+      url: process.env['EMAIL_SERVER'],
+    })
+  : {
+      async sendMail(options: SendMailOptions): Promise<void> {
```

Comment:
I like this a lot. Opening ethereal.email just to login in my local env was becoming annoying.